src/botWorker.js

```diff
@@ -0,0 +1,43 @@
+import dotenv from 'dotenv'
+dotenv.config({ path: '.env.local' })
+
+const { generateComment } = await import('./commentGenerator.js')
+
+console.log(process.env.SUPABASE_URL)
+import { createClient } from './lib/supabase/client.js'
+
+const supabase = createClient()
+
+export async function processBotQueue() {
+  try {
+    const { data: queueItems, error } = await supabase
+      .from('bot_queue')
```

Comment:
Hi, can you provide me the table schema for this table, so that I can test this PR before merging?